components/api/src/main/java/com/hotels/styx/api/HttpRequest.java

```diff
@@ -349,8 +349,7 @@ public String toString() {
         return toStringHelper(this)
                 .add("version", version)
                 .add("method", method)
-                .add("uri", url)
-                .add("headers", headers)
+                .add("url", url)
```

Comment:
It was renamed because we convert it from a URI to a URL in the code (line 421 of HttpRequest) and the underlying object is a URL object.  Still as David and I discussed, it is a fairly trivial issue.  I've no problem with changing it back.

Reply:
reverted to url